app/src/main/java/org/schulcloud/mobile/models/file/SignedUrlRequest.kt

```diff
@@ -10,7 +10,7 @@ class SignedUrlRequest {
         val ACTION_PUT = "putObject"
```

Comment:
This class isn't used anymore and can be deleted

Reply:
The class SignedUrlRequest is needed for file upload, which is currently not included in dev, but I think the class will be needed for uploading submissions. But the companion object is unused, I'll remove it.